### ISSUE TITLE
"Interactive transcripts" should be bold

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-media-guide#192]
The wording "Descriptive transcripts" is both bold and italics but "Interactive transcripts" is only italics which makes the text hard to follow as being two different topics.

**Comment by shawna-slh** [comment]
hummm. The main audience for this resource (from [RA](https://www.w3.org/WAI/EO/wiki/Accessible_Media_Resource#Requirements_Analysis)) is those providing media. From that perspective, an "interactive transcript" **is not a transcript** -- it's the media player making use of the captions file.

It's covered is those pages:
* https://www.w3.org/WAI/media/av/captions/#interactive-transcripts-from-captions
* https://www.w3.org/WAI/media/av/player/#player-accessibility-functionality

We included it on the transcripts page too because someone might have heard "interactive transcript" and go looking for it on the transcripts page.

I think we don't want to bold it for those reasons -- it's not parallel to a descriptive transcript from a provider perspective.
 
current:
>_Interactive transcripts_ highlight text phrases as they are spoken. Users can select text in the transcript and go to that point in the video. This is a feature of the media player. It uses the captions file.
[image]

We could so something to make it more clear, though.

1. Perhaps put info that it's not a transcript first:
> _Interactive transcripts_ are a media player feature that uses the captions file you provide. The player highlights text phrases as they are spoken. Users can select text in the transcript area and go to that point in the video.
[image]

-- though I prefer to put what it is first.

2. or less with a link:
> _Interactive transcripts_ are a feature of the media player that uses the captions file, as shown in [Interactive Transcripts from Captions](https://www.w3.org/WAI/media/av/captions/#interactive-transcripts-from-captions).
[NO image]

-- but seems not worth making them to do another page for three sentences? And nice to have an image here. :-)

4. Maybe original order with links:
>_Interactive transcripts_ highlight text phrases as they are spoken. Users can select text in the transcript and go to that point in the video. This is a [media player feature](https://www.w3.org/WAI/media/av/player/#player-accessibility-functionality). It uses your [captions file](https://www.w3.org/WAI/media/av/captions/#interactive-transcripts-from-captions ).
[image]

5. or even more clear/blunt/spelled out:
> _Interactive transcripts_ are not separate transcripts. They are a media player feature that uses the captions file you provide. The player highlights text phrases as they are spoken. Users can select text in the transcript area and go to that point in the video.
[image]

Thoughts?

**Comment by MicheleAWilliams-A11y** [comment]
Admittedly I was just scanning the page while we were talking on the call so I was just looking at the pattern and thought the bold was missed (e.g., how we have bold definitions for audio description). Reading more closely, I'm not sure how to interpret the formatting: "Basic transcripts" is plain text at the top (so doesn't stand out), "Descriptive transcripts" is bold and italic so your eye goes straight there, and "Interactive transcripts" is italicized so you follow that second and think it's related.

If the content is not meant to talk about/define these types of transcripts, I wouldn't make these visual distinctions (or use them more uniformly). Also, for "Interactive transcripts", I would de-emphasize the wording if it's just meant to call out what video players do. For example, "Notably, video players can provide interactive transcripts...".
